packages/flutter_tools/lib/src/base/command_help.dart

```diff
@@ -38,158 +34,134 @@ class CommandHelp {
-  CommandHelpOption get c => _c ??= _makeOption(
+  late final CommandHelpOption c = _makeOption(
     'c',
     'Clear the screen',
   );
 
-  CommandHelpOption _d;
-  CommandHelpOption get d => _d ??= _makeOption(
+  late final CommandHelpOption d = _makeOption(
     'd',
     'Detach (terminate "flutter run" but leave application running).',
   );
 
-  CommandHelpOption _g;
-  CommandHelpOption get g => _g ??= _makeOption(
+  late final CommandHelpOption g = _makeOption(
     'g',
     'Run source code generators.'
   );
 
-  CommandHelpOption _h;
-  CommandHelpOption get h => _h ??= _makeOption(
+  late final CommandHelpOption h = _makeOption(
     'h',
     'Repeat this help message.',
   );
 
-  CommandHelpOption _i;
-  CommandHelpOption get i => _i ??= _makeOption(
+  late final CommandHelpOption i = _makeOption(
     'i',
     'Toggle widget inspector.',
     'WidgetsApp.showWidgetInspectorOverride',
   );
 
-  CommandHelpOption _o;
-  CommandHelpOption get o => _o ??= _makeOption(
+  late final CommandHelpOption o = _makeOption(
     'o',
     'Simulate different operating systems.',
     'defaultTargetPlatform',
   );
 
-  CommandHelpOption _p;
-  CommandHelpOption get p => _p ??= _makeOption(
+  late final CommandHelpOption p = _makeOption(
     'p',
     'Toggle the display of construction lines.',
     'debugPaintSizeEnabled',
   );
 
-  CommandHelpOption _q;
-  CommandHelpOption get q => _q ??= _makeOption(
+  late final CommandHelpOption q = _makeOption(
     'q',
     'Quit (terminate the application on the device).',
   );
 
-  CommandHelpOption _r;
-  CommandHelpOption get r => _r ??= _makeOption(
+  late final CommandHelpOption r = _makeOption(
     'r',
     'Hot reload. $fire$fire$fire',
   );
 
-  CommandHelpOption _s;
-  CommandHelpOption get s => _s ??= _makeOption(
+  late final CommandHelpOption s = _makeOption(
     's',
     'Save a screenshot to flutter.png.',
   );
 
-  CommandHelpOption _t;
-  CommandHelpOption get t => _t ??= _makeOption(
+  late final CommandHelpOption t = _makeOption(
     't',
     'Dump rendering tree to the console.',
     'debugDumpRenderTree',
   );
 
-  CommandHelpOption _v;
-  CommandHelpOption get v => _v ??= _makeOption(
+  late final CommandHelpOption v = _makeOption(
     'v',
     'Launch DevTools.',
   );
 
-  CommandHelpOption _w;
-  CommandHelpOption get w => _w ??= _makeOption(
+  late final CommandHelpOption w = _makeOption(
     'w',
     'Dump widget hierarchy to the console.',
     'debugDumpApp',
   );
 
-  CommandHelpOption _z;
-  CommandHelpOption get z => _z ??= _makeOption(
+  late final CommandHelpOption z = _makeOption(
     'z',
     'Toggle elevation checker.',
   );
 
-  CommandHelpOption _k;
-  CommandHelpOption get k => _k ??= _makeOption(
+  late final CommandHelpOption k = _makeOption(
     'k',
     'Toggle CanvasKit rendering.',
   );
 
-  CommandHelpOption _M;
-  CommandHelpOption get M => _M ??= _makeOption(
+  late final CommandHelpOption M = _makeOption(
```

Comment:
Seems like a reasonable use of `late final`

Reply:
Levelling up!